source/_integrations/google_sheets.markdown

```diff
@@ -20,7 +20,7 @@ The integration currently only has access to that one document that is created d
 ## Prerequisites
 
 You need to configure developer credentials to allow Home Assistant to access your Google Account.
-These credentials are not the same as the ones for [Google Calendar](https://www.home-assistant.io/integrations/google).
+These credentials are not the same as the ones for [Google Calendar](/integrations/google).
```

Comment:
I think they are the same as Nest though! I can run through the setup though and come back and update the docs.